### ISSUE TITLE
Can't install from source if I don't have a GPU? 

### ISSUE THREAD

**Issue by spk921** [tensorflow/tensorflow#71]
My os X: EI Captian version 10.11.1 
I tried install from the source. However, running "$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg" this make me error.  I got following error message. I think I don't have a GPU in my mac. Is there anyway that I can isnstall from source even though I don't have a GPU ? 

sangpil_kim@ ~/git/tensorflow $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
.......
INFO: Found 1 target...
INFO: From Executing genrule //third_party/gpus/cuda:cuda_config_check:
/bin/bash: touch: command not found
ERROR: /Users/sangpil_kim/git/tensorflow/third_party/gpus/cuda/BUILD:143:1: Executing genrule //third_party/gpus/cuda:cuda_config_check failed: bash failed: error executing command /bin/bash -c 'source tools/genrule/genrule-setup.sh; touch bazel-out/local_darwin-opt/genfiles/third_party/gpus/cuda/cuda.config': com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 127.
Target //tensorflow/tools/pip_package:build_pip_package failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 79.345s, Critical Path: 0.32s
-bash: tail: command not found
-bash: sed: command not found

**Comment by vincentvanhoucke** [comment]
I suspect the GPU is a red herring. Without more info, the fact that /bin/bash can't seem to be able to execute standard tools like 'touch', 'tail' or 'sed' suggests something is very weird about your system.

**Comment by booherbg** [comment]
The actual error is that the 'touch' command fails. Fix that and you should be ok?

**Comment by spk921** [comment]
Thanks, after that error my bash cracked so I just reinstalled hole system. I hope to know that there is a case who build from source which don't have a GPU in their labtop.

**Comment by mrry** [comment]
After reinstalling your OS, are you able to run the commands `/usr/bin/touch`, `/usr/bin/tail` and `/usr/bin/sed` from the Terminal? If so, this step in the installation should succeed.

To clarify: **it is possible** to build TensorFlow from source without a GPU.

**Comment by spk921** [comment]
I used Time Machine before reinstalling it and I found those commands works. I assume my errors are related with CUDA configuration. I don't have a GPU but I installed CUDA so TensorFlow act like I have a GPU and makes a problem. However, I know it's possible to build without GPU. I will try to solve it next time when I have a time. I saw a benchmark that TensorFlow is not fast as torch so I decided to stick with Torch. I will try to install TensorFlow next time when it get better in the future.
